src/auth/authService.ts

```diff
@@ -18,6 +25,7 @@ export class AuthService {
     email: string,
     password: string
   ) {
+    const code = randomInt(100000, 999999);
```

Comment:
you can move this to other function such as `generateCode()`. So u can reuse this function in any places u need
(reset password etc).